draft-reddy-tls-slhdsa.md

```diff
@@ -156,6 +158,14 @@ The corresponding end-entity certificate when negotiated MUST use id-slh-dsa-sha
 
 The schemes defined in this document MUST NOT be used in TLS 1.2 {{RFC5246}}. A peer that receives ServerKeyExchange or CertificateVerify message in a TLS 1.2 connection with schemes defined in this document MUST abort the connection with an illegal_parameter alert.
 
+# SLH-DSA Variant Selection Guidance
+
+When deploying SLH-DSA in TLS 1.3, the choice of variant involves trade-offs among signing speed, verification cost, signature size, and the underlying hash function. The decision depends heavily on the characteristics and constraints of the target environment. If SHAKE is supported and offers acceptable performance, SHAKE-based variants are generally recommended. In environments where SHAKE is unavailable or performs poorly, SHA2-based variants are a suitable alternative.
```

Comment:
I'm not sure if I agree with this fully.  When the security of SHAKE is pretty solid, so is the security of SHA-2.  I wouldn't pick wording that would suggest otherwise.